Bembel/HomogenisedLaplace

```diff
@@ -0,0 +1,25 @@
+// This file is part of Bembel, the higher order C++ boundary element library.
+// It was written as part of a cooperation of J. Doelz, H. Harbrecht, S. Kurz,
```

Comment:
Replace with new header including the copy right.

Reply:
Done